pkg/installmanager/installmanager.go

```diff
@@ -368,8 +371,16 @@ func (m *InstallManager) Run() error {
 		m,
 		func(provision *hivev1.ClusterProvision) {
 			provision.Spec.Metadata = &runtime.RawExtension{Raw: metadataBytes}
-			provision.Spec.InfraID = pointer.StringPtr(metadata.InfraID)
-			provision.Spec.ClusterID = pointer.StringPtr(metadata.ClusterID)
+			if fakeCluster {
+				// If we're faking this install, it's safer to put an ID in that will never match any
+				// resource tags in the real world.
+				fakeClusterString := "fake cluster"
+				provision.Spec.InfraID = pointer.StringPtr(fakeClusterString)
+				provision.Spec.ClusterID = pointer.StringPtr(fakeClusterString)
+			} else {
+				provision.Spec.InfraID = pointer.StringPtr(metadata.InfraID)
+				provision.Spec.ClusterID = pointer.StringPtr(metadata.ClusterID)
+			}
```

Comment:
super not important:
A:
// set the most used path
// if fake, set the less taken code-path
B:
// if fake do this
// if not do the actual work

I personally like A .

Reply:
Changed to if !fakeCluster do the normal, else do the fake.